pidfile/pidfile_test.go

```diff
@@ -0,0 +1,61 @@
+// +build !windows
```

Comment:
For which reason this test is skipped on windows?
`Create` and `Remove` is performed in windows too, so it's better test them on windows